types/async-retry/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/async-retry",
-    "version": "1.4.9999",
+    "version": "1.3.9999",
```

Comment:
Backwards is unfortunately not a good direction to go; installing from npm will just keep people pulling 1.4.

Maybe we could manually deprecate all of the 1.4 releases, but anyone who installed those old packages will never get the new updates either.

I sort of think it's better to just leave this as-is...